src/strings/PowerFxResources.en-US.resx

```diff
@@ -3291,21 +3291,41 @@
     <comment>function_parameter - Second argument to the Patch function - the updates to be applied to the given rows.</comment>
   </data>
   <data name="AboutRemove" xml:space="preserve">
-    <value>Removes a specific record or records from a data source</value>
+    <value>Removes (optionally All) items from the specified 'collection'.</value>
     <comment>Description of 'Remove' function.</comment>
   </data>
-  <data name="RemoveDataSourceArg" xml:space="preserve">
-    <value>data_source</value>
-    <comment>function_parameter - First parameter for the Remove function. The data source that contains the records that you want to remove from. Translate this string. When translating, maintain as a single word (i.e., do not add spaces).</comment>
+  <data name="RemoveArg1" xml:space="preserve">
+    <value>collection</value>
```

Comment:
>collection [](http://example.com/codeflow?start=11&length=10)

We can also use Remove to delete items from a connected data source (not only a collection)